.gitignore

```diff
@@ -0,0 +1,13 @@
+.DS_Store
+
+# Virtual environment for Python 3
+venv/*
+
+# Compiled Python
+*.pyc
+
+# Example scripts and EAs
+extension_attributes/*
```

Comment:
@homebysix This still allows tracking of sub folders, correct? Is the idea that this line will keep errant files from being tracked within extension_attributes and scripts?

Reply:
Yes — line 10 excludes extension attributes, but line 11 re-includes the example EAs in the repo. Same for the scripts in lines 12-13.

This change is primarily for the benefit of those who want to contribute to the development of git2jss without having to manually remove downloaded scripts and extension attributes from the repo before committing/pushing.

Alternatively, I thought the ability to specify an alternate destination for the extension attributes and scripts would be a nice feature-add, but didn't have time yet to implement it.